huntsman/camera/pyro.py

```diff
@@ -450,30 +450,30 @@ def readout_time(self):
         return self._camera.readout_time
 
     @property
-    def ccd_temp(self):
-        temperature = self._camera.ccd_temp
+    def temperature(self):
+        temperature = self._camera.temperature
         return temperature.to(u.Celsius).value
 
     @property
-    def ccd_set_point(self):
-        temperature = self._camera.ccd_set_point
+    def target_temperature(self):
+        temperature = self._camera.target_temperature
         return temperature.to(u.Celsius).value
 
-    @ccd_set_point.setter
-    def ccd_set_point(self, set_point):
-        self._camera.ccd_set_point = set_point
+    @target_temperature.setter
+    def target_temperature(self, set_point):
```

Comment:
Not a big deal, but for consistency.

```suggestion
    def target_temperature(self, target):
```